media_utils/video_puppeteer.py

```diff
@@ -15,31 +15,14 @@ class VideoPuppeteer(object):
     Wrapper around HTML5 video element
     """
 
-    # Adapted from
-    # https://github.com/gavinsharp/aboutmedia/blob/master/chrome/content/aboutmedia.xhtml
-    _debug_script = """
-    var mainWindow = window.QueryInterface(Components.interfaces.nsIInterfaceRequestor)
-        .getInterface(Components.interfaces.nsIWebNavigation)
-        .QueryInterface(Components.interfaces.nsIDocShellTreeItem)
-        .rootTreeItem
-        .QueryInterface(Components.interfaces.nsIInterfaceRequestor)
-        .getInterface(Components.interfaces.nsIDOMWindow);
-    var tabbrowser = mainWindow.gBrowser;
-    for (var i=0; i < tabbrowser.browsers.length; ++i) {
-      var b = tabbrowser.getBrowserAtIndex(i);
-      var media = b.contentDocumentAsCPOW.getElementsByTagName('video');
-      for (var j=0; j < media.length; ++j) {
-         var ms = media[j].mozMediaSourceObject;
-         if (ms) {
-           debugLines = ms.mozDebugReaderData.split(\"\\n\");
-           return debugLines;
-         }
-      }
-    }"""
-
-    def __init__(self, marionette, url, video_selector='video'):
+    def __init__(self, marionette, url, video_selector='video', interval=1,
+                 duration=None, stall_wait_time=3, timeout=None):
         self.marionette = marionette
         self.test_url = url
+        self.interval = interval
+        self.stall_wait_time = stall_wait_time
+        self.timeout = timeout
```

Comment:
Added comment about usage in the doc string. Fixed usage so that if it is not specified, it will be 30 seconds. This is the timeout for the video element to appear.